desk/app/groups.hoon

```diff
@@ -1424,6 +1424,32 @@
         ==
       =.  cor  (emit (pass-hark & & yarn))
       go-core
+    ::
+        %edit
+      =/  prev=channel:g  (got:by-ch ch)
+      =/  =zone:g  zone.channel.diff
+      =.  zones.group
+        %+  ~(jab by zones.group)  zone
+        |=(=realm:zone:g realm(ord (~(push of ord.realm) ch)))
+      =.  channels.group  (put:by-ch ch channel.diff)
+      ?:  from-self  go-core
+      =/  link  (go-link /channels)
+      =/  yarn
+        %-  spin
+        :*  (go-rope /channel/edit)
+            link
+            `['Subscribe to channel' link]
+            ?:  &(!=(~ title.meta.prev) !=(title.meta.channel.diff title.meta.prev))
```

Comment:
Question for the Hoon Crew: what's a more idiomatic approach to express this conditional?